docs/.vuepress/local-components/landing/Header.vue

```diff
@@ -6,9 +6,9 @@
     </div>
     <div class="container mx-auto grid grid-cols-1 md:grid-cols-12 gap-4 mt-16 mb-2 px-6">
       <div class="md:col-span-7 lg:col-span-5 flex flex-col justify-center sm:text-center md:text-left landing-header-text">
-        <h1>Write in JavaScript.<br>Run native everywhere.</h1>
+        <h1>Write in JavaScript.<br>Run native everywhere.</h1> 
```

Comment:
What's at the end of the line?

Reply:
Dunno. Maybe a formatting space. Will remove it

Reply:
There's nothing I can remove

<img width="580" alt="Bildschirmfoto 2022-05-22 um 08 36 07" src="https://user-images.githubusercontent.com/1126641/169682304-bf9c159c-38a1-4882-b2c8-cb044c8f53b2.png">

Reply:
fixed